bin/get-report.r

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env Rscript
```

Comment:
I'd maybe add to the commit message here that this will only work if you have R installed, and indicate what R packages you need to have installed in R, e.g. at least `install.packages('rgdal')`, `install.packages('cleangeo')` - I can't find my notes on exactly what you need at the moment, but you get the idea!